Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -7521,6 +7522,72 @@
+        <Sight value="5"/>
+        <Food value="-1"/>
+        <CostCategory value="Army"/>
+        <CostResource index="Minerals" value="100"/>
+        <AttackTargetPriority value="10"/>
+        <DamageDealtXP value="1"/>
+        <DamageTakenXP value="1"/>
+        <KillXP value="5"/>
+        <Radius value="0.375"/>
+        <SeparationRadius value="0.375"/>
+        <InnerRadius value="0.375"/>
+        <ScoreResult value="BuildOrder"/>
+        <SubgroupPriority value="5"/>
+        <FlagArray index="UseLineOfSight" value="1"/>
+        <FlagArray index="NoScore" value="1"/>
+        <FlagArray index="AILifetime" value="1"/>
+        <PlaneArray index="Ground" value="1"/>
+        <Collide index="Ground" value="1"/>
+        <Collide index="ForceField" value="1"/>
+        <Collide index="Small" value="1"/>
+        <AbilArray Link="stop"/>
+        <AbilArray Link="move"/>
+        <AbilArray Link="que1"/>
+        <AbilArray Link="AP_MorphlingToBaneling"/>
+        <AbilArray Link="AP_MorphlingToRavager"/>
+        <AbilArray Link="AP_MorphlingToImpaler"/>
+        <AbilArray Link="AP_MorphlingToLurker"/>
+        <AbilArray Link="AP_MorphlingToBroodLord"/>
+        <AbilArray Link="AP_MorphlingToViper"/>
+        <AbilArray Link="AP_MorphlingToGuardianSCBW@Morph"/>
+        <AbilArray Link="AP_MorphlingToDevourerSCBW@Morph"/>
+        <CardLayouts>
+            <LayoutButtons Face="Move" Type="AbilCmd" AbilCmd="move,Move" Row="0" Column="0"/>
+            <LayoutButtons Face="Stop" Type="AbilCmd" AbilCmd="stop,Stop" Row="0" Column="1"/>
+            <LayoutButtons Face="MoveHoldPosition" Type="AbilCmd" AbilCmd="move,HoldPos" Row="0" Column="2"/>
+            <LayoutButtons Face="MovePatrol" Type="AbilCmd" AbilCmd="move,Patrol" Row="0" Column="3"/>
+            <LayoutButtons Face="AP_MorphlingToBaneling" Type="AbilCmd" AbilCmd="AP_MorphlingToBaneling,0" Row="1" Column="0"/>
+            <LayoutButtons Face="AP_Ravager" Type="AbilCmd" AbilCmd="AP_MorphlingToRavager,0" Row="1" Column="1"/>
+            <LayoutButtons Face="AP_Lurker" Type="AbilCmd" AbilCmd="AP_MorphlingToLurker,0" Row="1" Column="2"/>
+            <LayoutButtons Face="AP_MorphlingToImpaler" Type="AbilCmd" AbilCmd="AP_MorphlingToImpaler,0" Row="1" Column="3"/>
+            <LayoutButtons Face="BroodLord" Type="AbilCmd" AbilCmd="AP_MorphlingToBroodLord,0" Row="2" Column="0"/>
+            <LayoutButtons Face="AP_Viper" Type="AbilCmd" AbilCmd="AP_MorphlingToViper,0" Row="2" Column="1"/>
+            <LayoutButtons Face="AP_GuardianSCBW" Type="AbilCmd" AbilCmd="AP_MorphlingToGuardianSCBW@Morph,0" Row="2" Column="2"/>
+            <LayoutButtons Face="AP_DevourerSCBW" Type="AbilCmd" AbilCmd="AP_MorphlingToDevourerSCBW@Morph,0" Row="2" Column="3"/>
+        </CardLayouts>
+        <EditorCategories value="ObjectType:Unit,ObjectFamily:Melee"/>
+        <GlossaryCategory value="Unit/Category/ZergUnits"/>
+        <HotkeyCategory value="Unit/Category/ZergUnits"/>

```

Comment:
hotkey and glossary categories should be AP_ZergUnits or whatever the category is called.